Created style guide markdown file

aireynexxx opened the pull request:
## Using this Pull Request Template
1. In the Description of Changes section below, please fill out the details requested
2. Create the pull request
3. Check the boxes in the Type of Change section that apply to your changes
4. Check the boxes in the Checklist Before Review section that apply
5. Assign at least two reviewers to review your pull request by clicking the settings icon next to Reviewers

## Description of Changes
Please document the following about this pull request:
### Identify the organization for which you completed this work
If this work is associated with a university course, please identify the university/course/semester (ex. MTU/MIS3200/Fall2023).
* List the organizaiton here: 

### Identify what team you are on
* List the team here:

### Identify the task, story, or GitHub issue associated with this work
* Identify the task/story/issue here:

### Identify the particular changes that you made for this pull request
* Describe the changes here: 

## Type of Change (check all that apply)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation addition/update (non-breaking change that adds documentation to the system)

## Checklist Before Review
- [ ] Do you want to keep your branch for future changes or delete it once it is reviewed?
- [ ] I have followed the git practices outlined [here](GitPractices.md) to ensure code conflicts are minimized
- [ ] My code follows the style guidelines of this project outlined [here](documentation/StyleGuide.md)
- [ ] Update any dependency files. eg.(requirements.txt)
- [ ] I have performed a self-review of my code. It runs without errors and follows the [Best Coding Practices](BestCodingPractices.md)
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] There are no API keys, passwords, usernames, or other secrets in the code. I am using .env files for all secrets.
- [ ] I have added tests that prove my fix is effective or that my feature works

aireynexxx committed:
Created style guide markdown file

jeffreywallphd reviewed (changes requested):
Please see the StyleGuide.md file in the guides folder of the main directory. I want to avoid having duplicate files in the repository to avoid confusion. If you feel like the guides and documentation folder should be merged or the StyleGuide.md file in the guides folder belongs in the documentation folder, please move that file into documentation.

aireynexxx committed:
Style guide

Removed old style guide from 'guides', created 'design' folder and a 'style_guide.md' in the folder. Created a style guide, going to continue on working on it throughout the semester.

aireynexxx committed:
Merge branch 'main' into aireynexxx-patch-3

aireynexxx committed:
Small adjustments for style guide

jeffreywallphd reviewed (approved):
This is great! Thanks for helping to centralize information and keep it up-to-date. This new documentation folder will be very helpful. In a future iteration, you may want to link this Style Guide in the main README documentation file.

jeffreywallphd committed:
Merge branch 'main' into aireynexxx-patch-3
